### DISCUSSION TITLE
WiiU Pro Controller support

### DISCUSSION THREAD

**Discussion by Roet-Ivar** [cathery/sys-con#24] [Controller requests]
WiiU Pro Controller support would be awesome!
There is currently dongles like the [Mayflash Magic-NS Wireless Controller Adaptor](http://www.mayflash.com/Products/NINTENDOWiiU/MAGIC-NS.html) that works for the Switch. But hey, we're here to get rid of those dongles right?

**Comment by cathery** [comment]
damn right

**Comment by dibas** [comment]
Wii U Pro Controller support would be great, but I am pretty sure that the USB port is for charging only. I'd be happy to stand corrected, but otherwise we'll have to implement support for Bluetooth connections first.

**Comment by t0mtee** [comment]
@cathery Please implement this!

**Comment by dephunk** [comment]
Not exactly the same but i bought this many years ago - [MayFlash Wii U to PC/PS3 Adaptor](http://www.mayflash.com/Products/NINTENDOWiiU/W009.html) and can confirm it works when set to X-input... So would any xinput adapter made for PC theoretically work?

**Comment by cucholix** [comment]
@cathery would you open a bounty for this issue? I’m happily would help to fund this feature

**Comment by cathery** [comment]
There's a link to my ko-fi at the bottom of my info page. Wii U pro is currently 3rd on my priority list and it's bound to get sorted eventually

**Comment by NerdOrb** [comment]
Just Curious, any updates on Wii U Pro Controller support?
